tests/util_test.py

```diff
@@ -105,6 +105,24 @@ def test_looking_up_nested_composite_keys(ea):
     assert lookup_es_key(record, 'Fields.ts.value') == expected
 
 
+def test_looking_up_arrays(ea):
+    record = {
+        'flags': [1, 2, 3],
+        'objects': [
+            {'foo': 'bar'},
+            {'foo': [{'bar': 'baz'}]},
+            {'foo': {'bar': 'baz'}}
+        ]
+    }
+    assert lookup_es_key(record, 'flags[]') == 1
```

Comment:
is it just grabbing the first one? that doesn't seem super intuitive to me. I would've expected "jq-esque" behavior where this returns all the objects.

Whatever you end up choosing for the end user interface / behavior semantics, can you add documentation?

Reply:
I've removed this example. I'd like to support jq style in the future but for now, this is sufficient. Documentation added.